llama-index-integrations/llms/llama-index-llms-sambanova/pyproject.toml

```diff
@@ -9,10 +9,10 @@ skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
 
 [tool.llamahub]
 contains_example = false
-import_path = "llama_index.llms.sambanovacloud"
+import_path = "llama_index.llms.sambanova"
```

Comment:
any reason to rename all the imports and names?

Reply:
Unfortunately, someone published under the name `llama-index-llms-sambanova` already, so unless we found them and got them to transfer the name to our account (probably not easy), thats not an option 

I would suggest either adding another `llama-index-llms-sambanovastudio` package. Or maybe there's some other alternative name that would work for both